lib/internal/worker.js

```diff
@@ -80,6 +80,15 @@ class Worker extends EventEmitter {
                                      'array',
                                      options.execArgv);
     }
+    let args;
+    if (options.args) {
+      if (!Array.isArray(options.args)) {
+        throw new ERR_INVALID_ARG_TYPE('options.args',
+                                       'array',
+                                       options.args);
+      }
+      args = options.args.map(String);
```

Comment:
In the doc it says that these should be `string`s but here you are not enforcing that but silently convert them to strings. Perhaps it'd be better to error if one of the args is not a `string`? Or at least correct the doc by saying Array of any that will be converted to strings.

Reply:
It's true that all items would be stringified. It doesn't mean we would expect non-strings in the worker threads. I followed the pattern in docs of `child_process.spawn`,  in which `args` accepts any items and would also be stringified yet documented as `string[]`.

Reply:
I've updated the document and claiming that items on the list would be stringified first. PTAL :)